rel/overlay/etc/default.ini

```diff
@@ -275,7 +309,10 @@ require_valid_user = false
 timeout = 600 ; number of seconds before automatic logout
 auth_cache_size = 50 ; size is number of cache entries
 allow_persistent_cookies = true ; set to false to disallow persistent cookies
-iterations = 10 ; iterations for password hashing
```

Comment:
We should decide if we'd want to move all settings from `couch_httpd_auth` (which can be moved) to `chttpd_auth` or not move any in this PR. It is odd to move just one one setting leave say `allow_persistent_cookies = true` behind.

Reply:
Moved back. I will create another PR to move options from [couch_httpd_auth] to [chttpd_auth].